java/src/main/java/gov/doc/ntia/sigmf/ext/global/sensor/Sensor.java

```diff
@@ -38,12 +39,20 @@ public class Sensor implements Serializable {
   @JsonProperty(value = "mobile", required = false)
   protected Boolean mobile;
 
+  @JsonProperty(value = "geolocation", required = false)
+  protected GeoJsonPoint geolocation;
+
   @JsonProperty(value = "location", required = false)
   protected Location location;
```

Comment:
Does this need to be marked as deprecated?

Reply:
Yes- I made this change in 428c90b34cba6e737e243726f62150644205481a